### ISSUE TITLE
Auto Layout in Code

### ISSUE THREAD

**Issue by sibeliusseraphini** [codepath/ios_guides#22]
There should be a guide to define Layouts using only code.

There is some libraries that simplify this task:
- Cartography - https://github.com/robb/Cartography
- SwiftBox - https://github.com/joshaber/SwiftBox
- FLKAutoLayout - https://github.com/floriankugler/FLKAutoLayout

**Comment by vrwim-bazookas** [comment]
Yes, but a clear statement that in almost all cases, one can get away with defining constraints in code and then activating and deactivating them accordingly (or changing the `constant` value)
